### PR TITLE
remove duplicate suffix from calls_total spanmetric

### PR DESCRIPTION
**Pull request by fredr**
#### PR Description 
Because of an [upstream fix](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2837/files), the metric have duplicate `_total` suffixes and comes out as `traces_spanmetrics_calls_total_total`

#### Notes to the Reviewer
I wanted to write a test for this as well, but didn't find a way of doing it, since the name is being created in the processor upstream. If you have any suggestions on a test, please let me know.

#### PR Checklist

- [x] CHANGELOG updated 
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by mapno — APPROVED** [review]
LGTM, thanks for the contribution again!

File: pkg/traces/remotewriteexporter/exporter.go [hunk 1]
```diff
@@ -161,25 +160,17 @@ func (e *remoteWriteExporter) handleHistogramIntDataPoints(app storage.Appender,
...
-		dataPoints := m.Sum().DataPoints()
-		if err := e.handleScalarIntDataPoints(app, m.Name(), counterSuffix, dataPoints); err != nil {
-			return err
-		}
-	case pdata.MetricDataTypeGauge:
-		dataPoints := m.Gauge().DataPoints()
-		if err := e.handleScalarIntDataPoints(app, m.Name(), noSuffix, dataPoints); err != nil {
-			return err
-		}
+	dataPoints := m.Gauge().DataPoints()
```
Comment (mapno) [inline pkg/traces/remotewriteexporter/exporter.go:163]: The data type check is necessary to correctly get the datapoints. While maybe spanmetrics only generated gauges (unsure), it's safer to keep it.

**Review by mapno — CHANGES_REQUESTED** [review]
Sorry, missed a change that I believe should be reverted.

File: pkg/traces/remotewriteexporter/exporter.go (hunk 1 above)
Comment (fredr) [reply pkg/traces/remotewriteexporter/exporter.go:163]: Ah, my bad, sorry, missed that they where different, thought the only diff where the prefix!

Comment (fredr) [reply pkg/traces/remotewriteexporter/exporter.go]: Fixed! Thanks for catching that!

**Review by mapno — APPROVED** [review]
Thanks :)

**Pull request merged**